prompt_toolkit/widgets/base.py

```diff
@@ -508,6 +509,22 @@ def _(event):
         def _(event):
             self.current_value = self.values[self._selected_index][0]
 
+        @kb.add(Keys.Any)
+        def _(event):
+            if six.binary_type(self.values[self._selected_index][1]).startswith(event.data):
```

Comment:
Hi @gpotter2,
Did you test this on Python 3? I think ``event.data`` is a ``str``, not ``bytes``. Normally this conversion is not needed.

Reply:
Hi! You were right, that was not needed, I thought remembering having troubles with unicode being retrieved on Python 2, but it's working as so. Just updated the code